Specifications/catalog/catalog.protocol.md

```diff
@@ -0,0 +1,131 @@
+target attributes. The target of an offer is the asset associated with the containing asset entry. If an entry is has no associated policies (hasPolicy does not contain any offer
+entries), the asset is does not have any usage control policy.
+
+### 3.2 Distributions
+
+An asset may contain 0..N [DCAT Distributions](https://www.w3.org/TR/vocab-dcat-3/#Class:Distribution). Each distribution must have at least one `DataService` which specifies where
+the distribution is obtained. Specifically, a `DataService` specifies the endpoint for initiating a `ContractNegotiation` and `AssetTransfer`.
+
+### 3.3 DataService
+
+A DataService may specify an IDS service endpoint such as a `Connector`.
+
+#### 3.3.1 ids:dataServiceType
+
+If the DataService refers to an IDS service endpoint, it must include the property `ids:dataServiceType`:
+
+| Category   | Description                                                                |
+|------------|----------------------------------------------------------------------------|
+| Definition | Specifies the IDS service type                                             |
+| Domain     | [dcat:DataService](https://www.w3.org/TR/vocab-dcat-2/#Class:Data_Service) |
+| Type       | xsd:string                                                                 |
+| Note       | The value of this field is left intentionally open for future extension.   |
+
+The following table lists well-know IDS endpoint types:
+
+| Value         | Description          |
+|---------------|----------------------|
+| ids:connector | A Connector endpoint |
+|               |                      |
+
+
+#### 3.3.2 dcat:servesDataset
+
+Note that the property `dcat:servesDataset` should be omitted from the `DataService` since `DataSets` are included as top-level entries. Clients are not required to process the
+contents of `dcat:servesDataset`.
+
+## 4 Technical Considerations
+
+### 4.1 Queries and Filter Expressions
+
+A _**catalog service**_ may support catalog queries or filter expressions as an implementation-specific feature. However, it is expected that query capabilities will be implemented
+by the consumer against the results of a `CatalogRequestMessage,` as the latter is an RDF vocabulary. Client-side querying can be scaled by periodically crawling provider catalog
+services, caching the results, and executing queries against the locally-stored catalogs.
+
+### 4.2 Replication Protocol
+
+The catalog protocol is designed to be used by federated services without the need for a replication protocol. Each consumer is responsible for issuing requests
+to 1..N catalog services, and managing the results. It follows that a specific replication protocol is not needed, or more precisely, each consumer replicates data from catalog
+services by issuing `CatalogRequestMessages`.
```

Comment:
That's sounds not very efficient for big catalogs. Do we need to think about a diff message, only exchanging the deltas?

Reply:
We can always add this once the base protocol is set since it will likely become complicated to do with DCAT.